perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -135,12 +141,37 @@ def _useBotoApi(storage_schema):
     A boolean indicates whether or not to use boto API.
   """
 
-  if storage_schema == 'gs' or storage_schema == 's3':
+  if storage_schema == 'gs':
+    return True
+  else:
+    return False
+
+
+def _useBoto3Api(storage_schema):
+  """Decides if the caller should use the boto3 api given a storage schema.
+
+  Args:
+    storage_schema: The schema represents the storage provider.
+
+  Returns:
+    A boolean indicates whether or not to use boto API.
+  """
+
+  if storage_schema == 's3':
     return True
   else:
     return False
 
 
+def _getS3():
```

Comment:
Please consider a better name for this function...

e.g., _getBoto3ResourceForS3()